docs/jwst/background_step/arguments.rst

```diff
@@ -27,3 +27,21 @@ control the sigma clipping, and are passed as arguments to the astropy
   Sets the minimum (faintest) magnitude limit to use when selecting sources
   from the WFSS source catalog, based on the value of `isophotal_abmag` in the
   source catalog. Defaults to ``None``.
+
+``--wfss_maxiter``
+  Only applies to Wide Field Slitless Spectroscopy (WFSS) exposures.
+  Sets the maximum number of iterations allowed for iterative outlier rejection
+  during determination of the reference background scaling factor. Defaults to 5.
+
+``--wfss_rms_stop``
+  Only applies to Wide Field Slitless Spectroscopy (WFSS) exposures.
+  If the percentage difference in the RMS of the background-subtracted image
+  between iterations is smaller than this value, stop the iterative outlier
+  rejection process.
+  Defaults to 0, i.e., do all iterations up to ``wfss_maxiter``.
```

Comment:
The default described here does not match the default value in the step spec - from the discussion on the ticket, it appears as though this wasn't concretely specified. I think it would be reasonable to set to this to 1, but if you think 0 is more intuitive, I would not object. We just need the two locations to match.

Reply:
This is an RMS stopping criterion, so only stopping when delta RMS is zero seems more intuitive to me.  1 could be a valid value, and it seems that INS wants to run up to the maxiter criterion by default.  But I will check where the default documented here does not equal the spec

Reply:
Yes I also noticed this discrepancy, where in the code the default is set to 1. For the NIRISS tests I've performed, a default of 1 or 0 makes no difference as in both cases the code reaches the default maxiter of 5. But to be safe it could be 0 rather than 1.